src/storage/service/memcached_client.go

```diff
@@ -0,0 +1,27 @@
+package service
+
+import (
+	"github.com/bradfitz/gomemcache/memcache"
+)
+
+type MemcachedClientInterface interface {
```

Comment:
nit: add docs to interface methods

Reply:
nit: this comment looks rudimentary, as it repeats the name of type, either add more explanatory comment or remove it.